client/packages.go

```diff
@@ -30,17 +30,26 @@ import (
 	"github.com/snapcore/snapd/snap"
 )
 
+// StoreAccount holds information about a store account.
+type StoreAccount struct {
+	ID          string `json:"id"`
+	Username    string `json:"username"`
+	DisplayName string `json:"display-name"`
+}
+
 // Snap holds the data for a snap as obtained from snapd.
 type Snap struct {
-	ID               string        `json:"id"`
-	Title            string        `json:"title,omitempty"`
-	Summary          string        `json:"summary"`
-	Description      string        `json:"description"`
-	DownloadSize     int64         `json:"download-size,omitempty"`
-	Icon             string        `json:"icon,omitempty"`
-	InstalledSize    int64         `json:"installed-size,omitempty"`
-	InstallDate      time.Time     `json:"install-date,omitempty"`
-	Name             string        `json:"name"`
+	ID            string        `json:"id"`
+	Title         string        `json:"title,omitempty"`
+	Summary       string        `json:"summary"`
+	Description   string        `json:"description"`
+	DownloadSize  int64         `json:"download-size,omitempty"`
+	Icon          string        `json:"icon,omitempty"`
+	InstalledSize int64         `json:"installed-size,omitempty"`
+	InstallDate   time.Time     `json:"install-date,omitempty"`
+	Name          string        `json:"name"`
+	Publisher     *StoreAccount `json:"publisher,omitempty"`
+	// This to also the publisher's username for historic reasons.
```

Comment:
to  make it clear that the comment refers to the line below as usual, and not to the line above as one might assume because it says 'publisher', maybe change the comment to start with something like 'Developer' or 'The next line' or even 'Developer, below,' instead of 'This'.

Reply:
this came from mvo branch,  it's also not grammatical which I just noticed

Reply:
Indeed, my bad, sorry.